### PR TITLE
LL-2277 ~ LL-2300

### PR DESCRIPTION
**Pull request by LFBarreto**
Updated transaction details for :deciduous_tree: 

### Type

Feature

### Context

LL-2277 ~ LL-2300

### Parts of the app affected / Test plan

Updated :deciduous_tree: operations details look into the associated issue LL-2300 for details

### CODE REVIEW

File: src/renderer/families/tron/Votes/index.js [hunk 1]
```diff
@@ -51,6 +51,19 @@ const useTronSuperRepresentatives = () => {
   return sp;
 };
 
+export const useFormattedVotes = (
+  votes: ?Array<any>,
+  superRepresentatives: ?Array<any>,
+): Array<any> => {
+  return votes
```
Comment (gre) [inline src/renderer/families/tron/Votes/index.js]: this could be useMemo ed?

Comment (LFBarreto) [reply src/renderer/families/tron/Votes/index.js]: I guess but this is more of a data formatter than a provider I'll rename this function sorry 😅

File: src/renderer/families/tron/Votes/index.js [hunk 2]
```diff
@@ -52,6 +52,17 @@ const useTronSuperRepresentatives = () => {
   return sp;
 };
 
+// @TODO move this to common
```
Comment (gre) [inline src/renderer/families/tron/Votes/index.js]: can we try to keep live-common up to date each time we have these new functions? could be used on mobile

Comment (LFBarreto) [reply src/renderer/families/tron/Votes/index.js]: i'll make a PR with this :+1:

**Pull request merged**